addons/hr_attendance/models/res_users.py

```diff
@@ -24,3 +25,15 @@ def SELF_READABLE_FIELDS(self):
             'last_check_out',
             'total_overtime'
         ]
+
+    def _clean_attendance_officers(self):
+        officer_group = 'hr_attendance.group_hr_attendance_officer'
```

Comment:
```suggestion
        officer_group = self.env.ref('hr_attendance.group_hr_attendance_officer')
```

Reply:
I would rather not call it here directly since if we don't have any officers to remove, we don't need to perform a self.env.ref call